about.html

```diff
@@ -119,6 +120,7 @@ <h2 class="previous-course_title">Our academic history</h2>
                 </li>
             </ul>
         </section>
+        
         <footer>
             <img class="footer_img" src="images/codersblack.png" alt="coders circle logo">
```

Comment:
[REQUIRED] Kindly ensure that the footer on your about page has a white background on mobile screen sizes and is dark on desktop. I mentioned the previous review